tests/gameResource.spec.js

```diff
@@ -72,209 +82,249 @@ describe("resource : gameResource", function() {
 
   // leagues
   describe(": leagues", function() {
-    it("should build a proper url to retrieve league data for a single league using a numeric game key", function() {
-      game.leagues(328, "328.l.34014", null);
+    it("should build a proper url to retrieve league data for a single league using a numeric game key", function(done) {
+      nock("https://fantasysports.yahooapis.com")
+        .get("/fantasy/v2/game/328/leagues;league_keys=328.l.34014?format=json")
+        .reply(200, require("./nock-data/gameLeagues").league);
+      game.leagues(328, "328.l.34014", done);
```

Comment:
I was going to try to limit the amount of changes I made to the tests to show I wasn't breaking anything, but promises that throw errors (which a lot did when we didn't return a valid response) log warning/errors when nothing observes the error, and I didn't want to leave the tests logging a bunch of errors.